lms/templates/exercises.html

```diff
@@ -27,7 +27,7 @@ <h1 id="exercises-head">{{ _('תרגילים') }}</h1>
           </div>
           {%- endif %}
           {%- if exercise.get('is_checked') is none %}
-            {% set details = {'page': 'send', 'icon': 'upload', 'text': _('שלח'), 'css': 'send', 'page_id': exercise['exercise_id']} %}
+            {% set details = {'page': 'send', 'icon': 'upload', 'text': _('שלח'), 'css': 'send', 'page_id': exercise['course_id']} %}
```

Comment:
Mmmmmmmm are you sure? I think it should be a combinations

(It's Hachana Le Masgan for future better uploading system)